projects/js-packages/publicize-components/src/hooks/use-publicize-config/index.js

```diff
@@ -1,17 +1,26 @@
 import {
 	getJetpackExtensionAvailability,
 	isUpgradable,
+	getJetpackData,
 } from '@automattic/jetpack-shared-extension-utils';
 import { useSelect, useDispatch } from '@wordpress/data';
 import { store as editorStore } from '@wordpress/editor';
 
 const republicizeFeatureName = 'republicize';
 
+/**
+ * Hook that provides various elements of Publicize configuration,
+ * whether it's enabled, and whether resharing is available.
+ *
+ * @returns { object } The various flags and togglePublicizeFeature,
+ * for toggling support for the current post.
+ */
 export default function usePublicizeConfig() {
 	const { togglePublicizeFeature } = useDispatch( 'jetpack/publicize' );
-	const { available: isRePublicizeFeatureAvailable } = getJetpackExtensionAvailability(
-		republicizeFeatureName
-	);
+	const sharesData = getJetpackData()?.social?.sharesData ?? {};
+	const isShareLimitEnabled = sharesData.is_share_limit_enabled;
```

Comment:
I wonder if this is a good candidate for the store.

Reply:
I've taken this the other way and put everything into initial state as it seemed simpler. Is there an advantage to using the store? It's a feature flag that we'll end up removing eventually.